### PR TITLE
Implementing entity 'cancel' functionality. Will return user to previ…

### PR DESCRIPTION
**Pull request by ghost**
…ous state (or dashboard if no previous state is available).

### CODE REVIEW

File: app/src/app/admin/abstractEntitiesController.ts [hunk 1]
```diff
@@ -86,6 +95,20 @@ namespace app.admin {
             });
         }
 
+        /**
+         * Returns the user to the previous state.
+         */
+        public cancel():void {
+
+            if(this.previousState.abstract) {
```
Comment (ghost) [inline app/src/app/admin/abstractEntitiesController.ts:103]: Is there a better way to check if the last state is invalid? E.g. The user clicked on a direct link for an article.

Comment (zakhenry) [inline app/src/app/admin/abstractEntitiesController.ts:103]: I suspect the `fromState` would just be null

Comment (zakhenry) [inline app/src/app/admin/abstractEntitiesController.ts:103]: Alternate method http://stackoverflow.com/a/25945003/515578

Comment (ghost) [inline app/src/app/admin/abstractEntitiesController.ts:103]: It's not null, it looks like this:

```
{name: "", url: "^", views: null, abstract: true}
```

Check if the name is empty?

**Pull request merged**